tutorials/ph-ug-temp/ph-ug-temp.js.txt

```diff
@@ -0,0 +1,56 @@
+// Objectives: We want to pull Land Surface Temperature (LST) day data 

```

Comment:
Please rename to `.js` (remove `.txt` extension) and make sure the code is synced with the tutorial. Thanks!

Reply:
Done